packages/fxa-auth-server/scripts/recorded-future/check-and-reset.ts

```diff
@@ -77,36 +89,82 @@ Dry run mode is on.  It is the default; use '--dry-run false' when you are ready
 
 Domain: ${searchDomain}
 Filter: first_downloaded_date_gte=${firstDownloadedDateIsoString}
-Limit: 500
+Limit: ${resultsPerPageLimit}
 `);
     return 0;
   }
 
-  const res = await emitStatsdMetrics(
-    async () =>
-      await client.POST('/identity/credentials/search', {
-        body: {
-          domains: [searchDomain],
-          filter: {
-            first_downloaded_gte: firstDownloadedDateIsoString,
-          },
-          limit: 500,
-        },
-      }),
+  const payload = {
+    domains: [searchDomain],
+    filter: {
+      first_downloaded_gte: firstDownloadedDateIsoString,
+    },
+    limit: resultsPerPageLimit,
+  };
+  const _credentialsSearchFn = createCredentialsSearchFn(client);
+  const credentialsSearch = emitStatsdMetrics(
+    _credentialsSearchFn,
     'recorded-future.identity.credentials-search',
     statsd
-  )();
+  );
+  const credentials = await fetchAllCredentialResults(
+    credentialsSearch,
+    payload
+  );
+  statsd.increment(
+    'recorded-future.identity.credentials-search.results.total',
+    credentials.length
+  );
+
+  const emailLoginIdentities = credentials.filter(isLoginAnEmailAddress);
```

Comment:
Dear Reviewer, please ignore this nonsense on filtering for email address logins.  In the next PR I'll use the API's search filter to limit the results to email logins.